iceoryx_utils/source/posix_wrapper/timer.cpp

```diff
@@ -61,13 +61,6 @@ void Timer::OsTimer::callbackHelper(sigval data)
     auto index = Timer::OsTimerCallbackHandle::sigvalToIndex(data);
     auto descriptor = Timer::OsTimerCallbackHandle::sigvalToDescriptor(data);
 
-    /// @todo cxx::expect
```

Comment:
I know there is a warning, but it's just because `index` is currently a `uint8_t`. It could be changed in the future and then this check is necessary. Please cast `index` to an `uint32_t` instead

Reply:
done